ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -2935,7 +2932,7 @@ let event_wait_gen rpc session_id classname record_matches =
     (* true if anything matches now *)
     let find_any_match recs =
       let ls = List.map record_matches recs in
-      Option.is_some (List.find_opt (fun x -> x) ls)
+      List.exists Fun.id ls
     in
     find_any_match all_recs
```

Comment:
Just saw the lines above, this can be reduced to a simple:
```suggestion
   List.exists record_matches all_recs
```

The function around this needs simplification, but that's not for this fix